### ISSUE TITLE
console text uses only one color. why not use multiple colors?

### ISSUE THREAD

**Issue by Rellotscrewdriver** [Rellotscrewdriver/Digital-Measurement-Calculator#10]
current UI looks bland. it needs some colors to beautify
here's the code if you don't know about implementing colors in console
works with windows only

```
/* you can use these constants
FOREGROUND_BLUE
FOREGROUND_GREEN
FOREGROUND_RED
FOREGROUND_INTENSITY
BACKGROUND_BLUE
BACKGROUND_GREEN
BACKGROUND_RED
BACKGROUND_INTENSITY
*/

HANDLE hConsole = GetStdHandle(STD_OUTPUT_HANDLE);
SetConsoleTextAttribute(hConsole, FOREGROUND_BLUE | FOREGROUND_GREEN | FOREGROUND_INTENSITY);
std::cout << "I'm cyan! Who are you?" << std::endl;
```

**Comment by Rellotscrewdriver** [comment]
maybe I'll try it once I finish this issue #6

**Comment by Rellotscrewdriver** [comment]
unfortunately, the escape codes doesn't work in windows for some reason, even if I tried to run it with a public library.
so I HAVE to depend on Windows API when working on it. maybe I will research more about it
![image](https://user-images.githubusercontent.com/50903223/145757685-fa6e47b7-6fe8-420d-a8fb-c5e5a9a82918.png)

**Comment by Arsenic-ATG** [comment]
> unfortunately, the escape codes doesn't work in windows for some reason, even if I tried to run it with a public library.
> so I HAVE to depend on Windows API when working on it. maybe I will research more about it
> ![image](https://user-images.githubusercontent.com/50903223/145757685-fa6e47b7-6fe8-420d-a8fb-c5e5a9a82918.png)

It's not an OS problem, and more of your command line interpreter's problem 
iirc you use win 7 which uses [`cmd`](https://en.wikipedia.org/wiki/Cmd.exe) as its default CLI, which have quite a reputation of not being nice with ANSI support. 
But you can always use stuff like [ANSICON](https://github.com/adoxa/ansicon) to enable it there also.

**Comment by Rellotscrewdriver** [comment]
I've decided to use both ANSI and WIN32 API to show colors on separate OSes as ansicon is great for me to test the ANSI sequences while not being on Mac or Linux. but releasing it like that would require people to use ansicon in order to see the colors in the terminal(for windows users until Windows 10 build 16257 (1709)) which wouldn't be convenient. Hence, the desicion was to take both into account and use either of them when on different OSes. but Unfortunately, I don't have access to mac, windows 10 build 16257 or Linux(using it in a VM is laggy to a point it's unbearable) to test whether it works or not.
so could you help me on that? 


here's the example code to show you how am I going to do that:
```cpp
#ifdef _WIN32
void Colors::Red(std::string text){
 HANDLE xhConsole = GetStdHandle(STD_OUTPUT_HANDLE);
 SetConsoleTextAttribute(xhConsole, FOREGROUND_BLUE | FOREGROUND_GREEN | FOREGROUND_INTENSITY);
 std::cout << text << std::endl;
}
#else
void Colors::Red(std::string text){
 cout << "\x1B[31m" << text << "\033[0m" << endl;
}
#endif
```

**Comment by Arsenic-ATG** [comment]
> **Hey, If you're wondering what I'm upto nowadays and what I have decided to do**
> 
> I've decided to use both ANSI and WIN32 API to show colors on separate OSes as ansicon is great for me to test the ANSI sequences while not being on Mac or Linux. but releasing it like that would require people to use ansicon in order to see the colors in the terminal(for windows users until Windows 10 build 16257 (1709)) which wouldn't be convenient. Hence, the desicion was to take both into account and use either of them when on different OSes. but Unfortunately, I don't have access to mac, windows 10 build 16257 or Linux(using it in a VM is laggy to a point it's unbearable) to test whether it works or not. so could you help me on that?

sure, just ping me when you want the patch/revision to be tested

**Comment by Rellotscrewdriver** [comment]
so I made some changes to my decisions, instead of making my library to have. I used a third party library called [Termcolor](https://github.com/ikalnytskyi/termcolor), which made me to learn about submodules. I'm pretty confident that the third party library had already tested it on all the platforms so I don't think I need someone to test. 

I apologize for not telling you earlier as I might have reverted/changed my decision.
